package/kedro_viz/server.py

```diff
@@ -488,10 +488,42 @@ def _get_dataset_node(node_id, namespace):
 
 @app.route("/api/main")
 def nodes_json():
-    """Serve the pipeline data. This includes basic node data amongst others edges, tags, and layers."""
+    """Serve the data from all Kedro pipelines in the project.
+    This includes basic node data amongst others edges, tags, and layers.
+    """
     return jsonify(_DATA)
 
 
+@app.route("/api/pipelines/<string:pipeline_id>")
+def pipeline_data(pipeline_id):
```

Comment:
Should we also return 404 Json for invalid pipeline ID, similar to how it's done for `api/nodes/id`?

Reply:
👍 very good point